docs/commands/analytics.md

```
@@ -434,6 +452,28 @@ staqan-yt get-channel-analytics @yourchannel --output csv > demographics.csv
 - Subscribed vs not subscribed viewers
 - Performance by subscription status
 
+#### Columns
+
+Every report except `demographics` returns the same metric columns, in this
+order (#185):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)